assets/js/copy-code-blocks.js

```diff
@@ -0,0 +1,40 @@
+// copy-code-blocks.js
+// Copyright: 2023  Scientific Python Developers
+// Author: Adam Porter <adam@alphapapa.net>
+// License: BSD 3-Clause License
+//   see <https://raw.githubusercontent.com/scientific-python/scientific-python-hugo-theme/main/LICENSE>
+
```

Comment:
```suggestion
```

It's a standard practice on our projects not to include individual author affiliations.
The license file then applies to the entire project.

Reply:
Understood.  Thanks.

Because of the way Hugo concats local JS files together, I left the filename comment on the first and last lines, to make its contents easier to distinguish in the bundled source code.  But I can remove that too if you like.